### ISSUE TITLE
About training 

### ISSUE THREAD

**Issue by Xie-HIT** [castacks/tartanvo#18]
Thanks for your great work! However, I have some questions. I found training code is missing and I tried to re-implement it on KITTI. But the up-to-scale  loss function did not converge, do you encounter the same problem? And an another question: whether the performance is maintained if you run the training code several times? Thanks!

**Comment by shurandaa** [comment]
The same issue, but I tried to designed a loss function by myself according to the description in her paper. Training script is missing but you know...

**Comment by Zilong-L** [comment]
I am re-implementing on TartanAir Dataset and am currently getting these results for the flow-to-pose-only network at stage one. Does this seem correct to you?

![image](https://github.com/castacks/tartanvo/assets/56150579/92a62fc1-f64b-4a5d-8063-7eae1eacf795)

**Comment by Xie-HIT** [comment]
Similar to you. For my re-implementing, the network converged on TartanAir Dataset (only Hard or Easy&Hard both OK, and stage one converged very quikly).
However, the up-to-scale translation loss is oscillating when train from scratch on KITTI odometry benchmark. Now I suspect the oscillation is brought by moving objects and static scenes in KITTI, which do not exist in TartanAir. Unfortunately I did not go deeper...
